src/main/play-doc/intro/Install-EC2.md

```diff
@@ -0,0 +1,243 @@
+
+The resultant security groups should now have the following inbound rules
+
+SG-ELB Inbound Rules
+
+| Type    | Proto   | Port        | Source     | 
+| :------ | :-----  | :---------- | :--------- |
+| HTTP    | TCP     | 80          | 0.0.0.0/0  |
+| HTTPS   | TCP     | 443         | 0.0.0.0/0  |
+
+SG-Nodes Inbound Rules
+
+| Type	  | Proto  | Port        | Source     |
+| :------ | :----- | :---------- | :--------- |
+| Custom  |TCP     | 9005        | SG-ELB     |
+| Custom  |TCP     | 9999        | SG-ELB     |
+| Custom  |TCP     | 9004        | SG-Nodes   |
+| Custom  |TCP     | 10000-10999 | SG-Nodes   |
+| SSH     |TCP     | 22          | 0.0.0.0/0  |		
+
+#### Load Balancer
+
+Create a load balancer from the EC2 control panel. You will need to create an internet gateway and attach it to your VPC in order to have a public load balancer. We'll add an optional HTTPS protocol listener on port 443 to the default port 80 HTTP listener. For this tutorial we will map both of our listeners to instance port 9999. Add all three subnets to the load balancer and assign the load balancer to the SG-ELB security group. Optionally you can upload an SSL Certificate to use the ELB as your TLS endpoint if you added the HTTPS listener. For health monitoring we'll use ConductR's bundles call, HTTP:9005/bundles.
+
+### Preparing the AMI
+
+Launch a single instance of the desired base AMI to use as our image master. We'll use the Ubuntu 14.10 HVM EBS-SSD boot image in US-East-1, ami-ee793a86. If you choose another base image, use an EBS boot image as they are much easy to image. Be certain to assign a public ip address in instance details. 
+
+Access the console of the image instance with root access. For Ubuntu AMIs this is done as the user ubuntu using the PEM file specified at launch. The user ubuntu has sudo access. Other images will use different users. Check with the image provider for the correct user name to use.
+
+#### Installing JRE 8
+
+Install Java 8 as the default JRE. You will need to accept the Oracle license agreement.
+
+```bash
+sudo add-apt-repository -y ppa:webupd8team/java 
+sudo apt-get update 
+sudo apt-get -y install oracle-java8-installer && sudo apt-get clean
+sudo apt-get -y install oracle-java8-set-default
+echo "JAVA_HOME=/usr/lib/jvm/java-8-oracle" | sudo tee -a /etc/environment
```

Comment:
Could I get you to update the general installation page with instructions to install Java 8, or make this consistent with the general one. The general one presently reads:

> First ensure that you have the default-jre package installed (the default is JRE 7 at the time of writing):

> sudo apt-get install default-jre

...and so forth.

I was careful about that install procedure as I don't want to violate the official Oracle supported means of doing things. However given that your above approach requires the user to accept a license agreement then I think we're good.

We just need to be consistent. :-)